tokio/src/runtime/builder.rs

```diff
@@ -938,6 +941,25 @@ cfg_io_driver! {
             self.enable_io = true;
             self
         }
+
+        /// Enables the I/O driver and configures the capacity of events to be 
+        /// processed per tick.
+        ///
+        /// # Examples
+        ///
+        /// ```
+        /// use tokio::runtime;
+        ///
+        /// let rt = runtime::Builder::new_current_thread()
+        ///     .enable_io()
+        ///     .with_events_capacity(1024)
+        ///     .build()
+        ///     .unwrap();
+        /// ```
+        pub fn with_events_capacity(&mut self, capacity: usize) -> &mut Self {
```

Comment:
we don't use `with`, maybe try something like `max_io_events_per_tick` or something. Not perfect, but be descriptive.